.ci.yaml

```diff
@@ -1453,6 +1453,17 @@ targets:
       task_name: animated_image_gc_perf
     scheduler: luci
 
+  - name: Linux_android animated_complex_opacity_perf__e2e_summary
```

Comment:
We need an ownership entry in [animated_complex_opacity_perf__e2e_summary](https://github.com/flutter/flutter/blob/master/TESTOWNERS#L11)

Reply:
Done

Reply:
Uh, the suffix `.dart` is also needed to bypass the check.

Reply:
Okay now done :)